docs/sources/setup/install/helm/monitor-and-alert/with-grafana-cloud.md

```diff
@@ -19,13 +19,13 @@ This topic will walk you through using Grafana Cloud to monitor a Loki installat
 **Before you begin:**
 
 - Helm 3 or above. See [Installing Helm](https://helm.sh/docs/intro/install/).
-- A Grafana Cloud account and stack (including Cloud Grafana, Cloud Metrics, and Cloud Logs)
-- [Grafana Kubernetes Monitoring using Agent](/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/configuration/) configured for the Kubernetes cluster
-- A running Loki deployment installed in that Kubernetes cluster via the Helm chart
+- A Grafana Cloud account and stack (including Cloud Grafana, Cloud Metrics, and Cloud Logs).
+- [Grafana Kubernetes Monitoring using Agent Flow](/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/configuration/config-k8s-agent-flow) configured for the Kubernetes cluster.
```

Comment:
nit: Include the trailing slash to avoid a redirect to the Hugo pretty URL
```suggestion
- [Grafana Kubernetes Monitoring using Agent Flow](/docs/grafana-cloud/monitor-infrastructure/kubernetes-monitoring/configuration/config-k8s-agent-flow/) configured for the Kubernetes cluster.
```